### ISSUE TITLE
Inbox not sync with Nextcloud

### ISSUE THREAD

**Issue by Turgutz2024** [alainm23/planify#1162]
The Planify Inbox is not synchronising using Nextcloud.
I am using Ubuntu 22.04
Anybody knows this problem? Is there a solution?

**Comment by alainm23** [comment]
It could be, but what would the Inbox task list be? currently the Inbox view is a local internal project.

**Comment by privx-de** [comment]
A setting where tasks are created by default (inbox or a project) would be really cool. Inbox syncing to Nextcloud would also be an option. The current situation is not ideal, every time I create a task quickly and forget the project, I only have it locally on the computer.

**Comment by oktayacikalin** [comment]
I think, due to the fact that Nextcloud Tasks does not have an inbox, it should be possible to hide and disable it. I don't need it. I can hide the inbox, but it's still available when adding new tasks, which is confusing.

I would rather have the pinboard or today view as my default. As of now, I'm using the pinboard as my "home" view.

**Comment by Turgutz2024** [comment]
maybe if i could tell the program that one of my nextcloud-lists sync with the planify inbox? so that i can allocate it...

**Comment by oktayacikalin** [comment]
Yes, if you really like having an inbox, then mapping it with a list in nextcloud should be the right way.

**Comment by oktayacikalin** [comment]
Perhaps having the option to either having it locally, mapping it to nextcloud or disable it, would be nice to have?

How does it work when using todoist?

**Comment by HashCat03** [comment]
I am having the same issue with Syncing to Todoist. All categories sync flawlessly with todoist with one exception: when i add entries to my inbox they are not synced to Todoist. That is, if I tap the big blue "Inbox" Button. 
Planify also displays the Inbox folder from my Todoist account further down below in the sidebar. If i enter entries there, they are correctly synced to my todoist account as expected. So the program is keeping two separate folders: one local inbox and one Todoist inbox. It would be really helpful to have a setting to merge the two so that if i tap the big blue inbox button it also gets added to my todoist inbox.

**Comment by nicojx** [comment]
I agree that would be useful. If your standard approach is to use Planify with Nextcloud Tasks (as for me), then an Inbox that can't be integrated doesn't make sense. On Linux at least, you already have a setting that allows linking a global shortcut with a list -- it might make sense to just expand that setting and link the Inbox, too. The broad idea -- default tasks need a home -- seems the same to me.
